assignments/run_callback/run_callback.js

```diff
@@ -1,4 +1,6 @@
 function RunCallback(a, b, cb) {
+    result = a+b
```

Comment:
it's looks fine for the best practice use let or var to declare a variable

Reply:
sure